Dockerfile

```diff
@@ -11,6 +11,7 @@ ENV farmer_address="null"
 ENV farmer_port="null"
 ENV testnet="false"
 ENV full_node_port="null"
+ENV self_hostname="127.0.0.1"
```

Comment:
In what situations is 127.0.0.1 not working for self_hostname?

Reply:
Since this is used to set the address RPC services listen on, any metrics gathering using RPC interfaces will in-fact be coming from a sister container and not from within the container itself.